src/sentry/notifications/notifications/activity/release.py

```diff
@@ -164,7 +164,7 @@ def get_title_link(self, recipient: Team | User) -> str | None:
 
     def build_notification_footer(self, recipient: Team | User) -> str:
         # notification footer only used for Slack for now
```

Comment:
Can this comment be removed since it sounds like it's not true anymore?

Reply:
Thanks for pointing out, updated!